frontend/app/src/typing/types.ts

```diff
@@ -150,7 +150,10 @@ export interface Account {
 
 export interface DSRBalance {
   readonly address: string;
-  readonly balance: BigNumber;
+  readonly balance: {
```

Comment:
There should already be a type containing amount and usdValue can we use that instead?

Reply:
Good point. Will add.